sullivan_a05a.js

```diff
@@ -0,0 +1,38 @@
+/*  Anna Sullivan
+    sullivan_a05a.js
+    INFO2124WW
+    Thoendel
+    1/16/2020
+*/
+
+{
+
+let todoArray = ["Take out trash", "Clean dishes", "Mop floors", "Brush dog"]; /* Declared ToDo List Array literal syntax and populated with 4 items*/
+
+let groceryListArray = new Array("Cheese", "Milk", "Bread", "Beans", "Tomatos", "Turkey"); /* Declared Grocery List Array using constructor function and populated with 6 items*/
```

Comment:
I see that you handled all your declarations up front - which is nice! AND I can follow what you are doing because you used comments to document! Awesome! 🌟